### PR TITLE
Thellimist/pi 993

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

**Comment by thellimist** [comment]
@jwineman Merged for beta-testing

File: src/WordPress/HTTP2ServerPush.php [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?php
 
-// TODO: Get rid of $GLOBALS and use static variables
-namespace CF\Hooks;
-
-use CF\WordPress\Utils;
+// TODO: 
+// Get rid of $GLOBALS and use static variables
+// Make class functions non-static
+namespace CF\WordPress;
```
Comment (jwineman) [inline src/WordPress/HTTP2ServerPush.php]: @thellimist Is this resolved?

Comment (thellimist) [inline src/WordPress/HTTP2ServerPush.php]: It is used here: https://github.com/cloudflare/Cloudflare-WordPress/blob/c30fc3e478c9662b46e6fe883afae3d2d8b6ba38/src/WordPress/HTTP2ServerPush.php#L57

Also everything is static which is not how we want to write code. They are still TODO's.

Comment (thellimist) [inline src/WordPress/HTTP2ServerPush.php]: @jwineman

**Pull request merged**